addons/copy-message-link/profile.css

```diff
@@ -0,0 +1,7 @@
+#comments .comment .report {
+  display: block;
+  opacity: 0;
+}
+#comments .comment:hover .report {
+  opacity: 1;
+}
```

Comment:
Explain these changes...?

Reply:
`displayNoneWhileDisabled` wouldn't work if `display: none` is used to hide the buttons when not hovered.